README.rst

```diff
@@ -76,6 +72,37 @@ Feature extraction:
         <user_is_anon>: False
         <user_is_bot>: False
 
+Installation
+================
+In order to use this, you need to install a few packages first:
+
+``pip install revscoring``
+
+You'll need to download NLTK data in order to make use of language features.
+
+.. code-block:: python
+
+    >>> python
+    >>> import nltk
+    >>> nltk.download()
+    >>> Downloader> d
+    >>> Identifier> wordnet
+    >>> Downloader> d
+    >>> Identifier> omw
+    >>> Downloader> q
+    >>> exit()
+
+
+You might need to install some other dependencies depending on your operating
+system.  These are for ``scipy`` and ``numpy``.
```

Comment:
I mean, the comment says "g++ git python-sklearn python3.4-dev libatlas-base-dev gfortran" are for `scipy` and `numpy`, but you were able to install them before `scikit-learn`, so `python-sklearn` is not needed for them (but for sklearn itself).